src/legacy/ui/public/vislib/lib/dispatch.js

```diff
@@ -91,6 +91,33 @@ export function VislibLibDispatchProvider(Private, config) {
       };
     }
 
+    /**
+     * Determine whether rendering a series is configured in percentage mode
+     * Used to display a value percentage formatted in it's popover
+     *
+     * @param rawId {string} The rawId of series to check
+     * @param series {Array} Array of all series data
+     * @param config {VisConfig}
+     * @returns {Boolean}
+      */
+    isSeriesInPercentageMode(rawId, series, config) {
+      if(!rawId || !Array.isArray(series) || !config) {
+        return false;
+      }
+      //find the primary id by the rawId, that id is used in the config's seriesParams
+      const { id } = series.find(series => series.rawId === rawId);
+      if(!id) {
+        return false;
+      }
+      //find the matching seriesParams of the series, to get the id of the valueAxis
+      const { valueAxis } = config.get('seriesParams').find(param => param.data.id === id);
+      if(!valueAxis) {
+        return false;
+      }
+      const usedValueAxe = config.get('valueAxes').find(valueAxe => valueAxe.id === valueAxis);
```

Comment:
nit: `usedValueAxis`?

Reply:
and then we can name this valueAxis

Reply:
🎶 axe to axis,  funk to funky, you know Major Tom is 🎶